### PR TITLE
Inmate comps

### PR DESCRIPTION
**Pull request by stevetotheizz0**
Adding button and callout comps

### CODE REVIEW

**Review by alejandroxlopez — CHANGES_REQUESTED** [review]
We need to discuss this more deeply with the team, I recommend doing at least some changes pointed out in the file comments.

**Review by ibdf — CHANGES_REQUESTED** [review]
@stevetotheizz0 We need to submit each component on it's own PR, so that we can review them together during Pizza meeting.

**Review by stevetotheizz0 — COMMENTED** [review]
@ibdf Is there an easy way to separate changes out into separate PR's? Also, do any changes nee dot wait until the bi-weekly meeting to be merged?

**Pull request closed without merging**
